.github/workflows/make-release.yml

```diff
@@ -0,0 +1,107 @@
+name: 'Yakut application builder'
+
+# Builds Yakut release
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
```

Comment:
Can we please introduce the same conditional enablement as in the other projects? The logic should be as follows:

- Push to any branch should run the test job.
- Push to master OR if the commit message contains `#release` should run the release job. This is useful for pushing experimental releases like `v0.15-dev3` from development branches.

Reply:
I have split the workflow into two workflows, one for the test and one for the release. A workflow is defined by its trigger. Since the trigger for these two jobs is significantly different, it makes sense to put them into separate workflows.